.gitignore

```diff
@@ -5,6 +5,7 @@
 .submodule-init
 .mypy-cache
 .pytest-cache
+.python-version
```

Comment:
Seems like this file is getting generated somewhere for me. Assumed it was fine to just add to the gitignore